src/offlinelogstorage/dlt_offline_logstorage_behavior.c

```diff
@@ -912,6 +915,11 @@ DLT_STATIC void dlt_logstorage_close_file(DltLogStorageFilterConfig *config)
         gzclose(config->gzlog);
         config->gzlog = NULL;
     }
+    
+    if (config->log) {
```

Comment:
this code is now duplicated (see lines below the #endif" ). Please remove it from the commit as it will not have any effect